wireapp/kalium
# feat: add api version to authenticated network requests

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

add api version to authenticated network requests

### Solutions

adding the API version to request path only when the version is >= 1

Needs releases with:

#507 


### Testing

#### Test Coverage (Optional)


#### How to Test

1. unit test.
2. run CLI and check the URL in the network logs it should have the API version.

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
